Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalModuleAlgo.cc

```diff
@@ -30,32 +30,32 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
   const auto& wafer = args.value<std::vector<std::string> >("WaferName");    // Wafers
   auto materials = args.value<std::vector<std::string> >("MaterialNames");   // Materials
   const auto& names = args.value<std::vector<std::string> >("VolumeNames");  // Names
-  const auto& thick = args.value<std::vector<double> >("Thickness");         // Thickness of the material
+  auto thick = args.value<std::vector<double> >("Thickness");                // Thickness of the material
```

Comment:
why is this now a copy rather than const ref? It doesn't seem like the values are changed anywhere later.

Reply:
put back